src/documentation/templates/samples-index.hbs

```diff
@@ -302,13 +301,13 @@
         </div>
         <div class="od-OfficeFooter-secondary">
           <ul class="od-OfficeFooter-list od-OfficeFooter-list--secondary">
-            <li><a href="http://www.microsoft.com/info/privacy.mspx" target="_blank">Privacy</a></li>
+            <li><a href="https://go.microsoft.com/fwlink/?LinkId=512132" target="_blank">Privacy</a></li>
             <li><a href="http://msdn.microsoft.com/en-US/cc300389" target="_blank">Terms of use</a></li>
             <li><a href="http://www.microsoft.com/library/toolbar/3.0/trademarks/en-us.mspx" target="_blank">Trademarks</a></li>
           </ul>
           <div class="od-OfficeFooter-microsoft">
             <img src="{{relativePath}}images/logo-microsoft.png" width="123" height="24" alt="Microsoft logo" />
-            <div>&copy; 2016 Microsoft</div>
+            <div>&copy; 2017 Microsoft</div>
```

Comment:
How often does the year change, really? Every couple of years? Seems like an over-optimization.